andela/hc-rogue-one
# Ft 156577840 nag user to resolve issue

#### What does this PR do?
Nag a user to resolve an issue like when a check is down and sends them notifications after a specified period
#### Description of Task to be completed?
The system should keep nagging the user every so often until they resolve the job that has gone down
#### How should this be manually tested?
1. Install the project and install the requirements.txt
2.Run the commands below
3.You should have send email configured 

cd to hc-rogue-one
``` python manage.py sendalerts```
in another terminal
``` python manage.py sendalerts```

#### What are the relevant pivotal tracker stories?
#156577840

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
